src/libraries/Microsoft.PowerFx.Core/IR/CoercionKind.cs

```diff
@@ -107,6 +107,7 @@ internal enum CoercionKind
         CurrencyToText,
         TextToCurrency,
         CurrencyToBoolean,
-        BooleanToCurrency,
+        BooleanToCurrency,
+        CurrencyToDecimal
```

Comment:
>CurrencyToDecimal [](http://example.com/codeflow?start=8&length=17)

Do we need a DecimalToCurrency conversion, if we need to send a number back to the table?